app/helpers/markup_helper.rb

```diff
@@ -0,0 +1,35 @@
+module MarkupHelper
```

Comment:
It is a thought.  But since there are only 1-line entries, a full Markdown syntax seems overkill.